src/pretix/presale/forms/organizer.py

```diff
@@ -0,0 +1,88 @@
+#
+# This file is part of pretix (Community Edition).
+#
+# Copyright (C) 2014-2020 Raphael Michel and contributors
+# Copyright (C) 2020-2021 rami.io GmbH and contributors
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation in version 3 of the License.
+#
+# ADDITIONAL TERMS APPLY: Pursuant to Section 7 of the GNU Affero General Public License, additional terms are
+# applicable granting you additional permissions and placing additional restrictions on your usage of this software.
+# Please refer to the pretix LICENSE file to obtain the full terms applicable to this work. If you did not receive
+# this file, see <https://pretix.eu/about/en/license>.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License along with this program.  If not, see
+# <https://www.gnu.org/licenses/>.
+#
+from django import forms
+from django.conf import settings
+from i18nfield.strings import LazyI18nString
+
+from pretix.base.models import EventMetaValue, SubEventMetaValue
+
+
+def meta_filtersets(organizer, event=None):
```

Comment:
In my test organizer, I do have lots of old events, that are live and accessible via the „old=1“ GET-param. Some of them have custom meta-data that newer events don’t. The filter shows these meta-data-values as options to filter for, but when I filter, I do not get any results as old events are not enabled to be shown. What would be the best way to handle this? Instead of automatically including old events if a filter does not yield any result, I would not add meta-data-values of events that happened in the past (if the old=1 GET param is not set).